docs/references/javascript/organization/invitations.mdx

```diff
@@ -11,13 +11,36 @@ import { CodeBlockTabs } from "@/components/CodeBlockTabs";
 
 These are all methods on [the Organization class](/docs/references/javascript/organization/organization) that allow you to manage the invitations to an organization.
 
-## `getPendingInvitations()`
+## `getInvitations()`
+
+Retrieves a list of invitations for the currently active organization.
 
 ```typescript
-function getPendingInvitations(params?: GetPendingInvitationsParams): Promise<OrganizationInvitation[]>;
+function getInvitations(params?: GetInvitationsParams): Promise<PaginatedResources<OrganizationInvitation>>;
 ```
 
-Retrieve the members of the selected organization.
+### `GetInvitationsParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | `number` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
+| `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
+| `status?` | `'pending' \| 'accepted' \| 'revoked'` | The status an invitation can have. |
+
+### Returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise{"<"}<a href="/docs/references/javascript/types/paginated-response">PaginatedResponse</a>{"<"}<a href="/docs/references/javascript/organization-invitation">OrganizationInvitation</a>{">"}</code> | This method returns a <code>Promise</code> which resolves with a list of <code><a href="/docs/references/javascript/organization-invitation">OrganizationInvitation</a></code> objects.|
```

Comment:
I would refactor to use reference links, like we do on /organization/domains 
 and /organization-domain

Don't forget to add the links at the bottom of the page ( [org-inv-ref]: /docs/references/javascript/organization-invitation)

```suggestion
| <code>Promise\<[PaginatedResponse][pag-res-ref]\<[OrganizationInvitation][org-inv-ref]\>></code> | This method returns a `Promise` which resolves with a list of [`OrganizationInvitation`][org-inv-ref] objects. |
```